compiler/AST/expr.cpp

```diff
@@ -1435,7 +1436,7 @@ GenRet codegenFieldPtr(
       // Must be a record or union type, and we must have an
       // lvalue-ptr to one of them.
       INT_ASSERT(isRecord(ct) || isUnion(ct));
-      INT_ASSERT( base.isLVPtr != GEN_VAL );
+      //INT_ASSERT( base.isLVPtr != GEN_VAL );
```

Comment:
Why comment out this assert? It seems to me codegenFieldPtr can't do it's job if base is a value. What case are you running in to?